<section class="section experiences-section">
                <h2 class="section-title"><i class="fa fa-briefcase"></i>Experiences</h2>

								<div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Scientific Director, Center for Genomics</h3>
                            <div class="time">2014 - present</div>
                        </div><!--//upper-row-->
                        <div class="company">Florida State University</div>
                    </div><!--//meta-->
                    <div class="details">
                        <p>Upon receiving my PhD, I accepted a position as Director of the FSU Center for Genomics because of my deep interest in utilizing diverse types of large-scale biological data to answer important questions in genome biology, including transcriptional regulation, genetic variation, chromatin dynamics, and replication timing. As the Director of a Center focused on genomic data analysis, I have led the analysis of genomic data for many projects, including those that have utilized RNA-seq, ChIP-seq, MNase-seq, Repli-seq, and other types of genomic data. </p>

                    </div><!--//details-->
                </div><!--//item-->

                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">PhD training</h3>
                            <div class="time">2010 - 2014</div>
                        </div><!--//upper-row-->
                        <div class="company">Florida State University</div>
                    </div><!--//meta-->
                    <div class="details">
                        <p>My graduate training involved the analysis of chromatin structure dynamics in humans and maize using enzymatic probes of chromatin structure coupled with microarrays and sequencing. It was during this time that I self-trained in data analysis and programming in order to analyze microarray and sequencing-based chromatin structure data. Through the process of self-training, I developed user-friendly software in R for analyzing and integrating different sources of genomic data sets, including ChIP-seq, RNA-seq, and MNase-seq data (http://github.com/dvera/travis). During my graduate studies, I also developed an approach to profiling multiple aspects of chromatin structure simultaneously (DOI). I also helped develop an approach to cost-effectively profile the chromatin landscapes in the human genome using targeted enrichment and illumina sequencing (DOI).</p>

                    </div><!--//details-->
                </div><!--//item-->
</section><!--//section-->
