<section class="section experiences-section">
                <h2 class="section-title"><i class="fa fa-briefcase"></i>Experiences</h2>

								<div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Scientific Director, Center for Genomics</h3>
                            <div class="time">2015 - present</div>
                        </div><!--//upper-row-->
                        <div class="company">Florida State University</div>
                    </div><!--//meta-->
                    <div class="details">
                        <p>As Director of the FSU Center for Genomics, I provide genomic data analysis services for researchers, run workshops on data generation and analysis, and consult on all aspects of genomic experiments, including experimental design. I lead the analysis of genomic data for many projects, including those that have utilized RNA-seq, ChIP-seq, MNase-seq, Repli-seq, de novo sequencing, and other types of data.</p>

                    </div><!--//details-->
                </div><!--//item-->

                <div class="item">
                    <div class="meta">
                        <div class="upper-row">
                            <h3 class="job-title">Graduate Research Assistant</h3>
                            <div class="time">2010 - 2014</div>
                        </div><!--//upper-row-->
                        <div class="company">Florida State University</div>
                    </div><!--//meta-->
                    <div class="details">
                        <p>My graduate training was under the mentorship of Hank Bass and Jonathan Dennis at FSU, and involved the analysis of chromatin structure dynamics in humans and maize using micrococcal nuclease as a probe of chromatin structure coupled with microarrays and sequencing. I developing an approach to profile nucleosome positioning, nucleosome sensitivity, and transcription factor binding simultaneously with MNase (DOI). I also was involved in the development of an approach to cost-effectively profile the chromatin landscapes in the human genome using targeted enrichment and illumina sequencing (DOI). Throughout the process of generating large-scale data sets, I self-trained in data analysis and programming in order to analyze the data I generated. As I learned to program, I developed user-friendly software in R to enable others to analyze and integrate different genomic data sets (http://github.com/dvera/travis).</p>
                    </div><!--//details-->
                </div><!--//item-->
</section><!--//section-->
